app/representation.py

```diff
@@ -185,7 +186,7 @@ def eventRecord(yelpId, lat, lon, title, startTime, endTime, url):
     isoEndTime = parser.parse(endTime) if endTime else isoStartTime + datetime.timedelta(hours=1)
 
     r = {
-            "id": title[:30].replace(" ", "-") + startTime,
+            "id": hashlib.md5(title[:30] + startTime).hexdigest(),
```

Comment:
nit: Add a comment that we:
1. are ensuring uniqueness so, when we update, old version of the events will be overwritten, rather than creating new events
2. avoiding "injection" into firebase parameters by hashing and specifically using the hexdigest (at least, I assume that's why we use it over `digest()`)

Might be worth making it a helper method, for comment clarity (and maybe tests for uniqueness/injection in the future ;).